pallet/src/lib.rs

```diff
@@ -32,16 +43,37 @@ pub mod pallet {
             + RuntimeAppPublic
             + Default
             + MaybeSerializeDeserialize;
+        type Event: From<Event<Self>> + IsType<<Self as frame_system::Config>::Event>;
+    }
+
+    #[pallet::event]
+    #[pallet::metadata(T::AccountId = "AccountId")]
+    #[pallet::generate_deposit(pub(super) fn deposit_event)]
+    pub enum Event<T: Config> {
+        ChangeValidators(Option<Vec<T::AccountId>>),
     }
 
+    pub struct AlephSessionManager<T>(sp_std::marker::PhantomData<T>);
+
     #[pallet::pallet]
     pub struct Pallet<T>(sp_std::marker::PhantomData<T>);
 
     #[pallet::hooks]
     impl<T: Config> Hooks<BlockNumberFor<T>> for Pallet<T> {}
 
+    //ensure_root
     #[pallet::call]
-    impl<T: Config> Pallet<T> {}
+    impl<T: Config> Pallet<T> {
+        #[pallet::weight(10_000)]
+        pub fn change_validators(
```

Comment:
shouldn't we also have another call to remove a validator?

Reply:
My comment was from operational perspective: in most situtations we will rather be adding or removing a single validator, and this way we do not need to keep track of a list of validators ourselves, making sure it's up to date (which can be error prone). YMMV, it's a non-blocker from my perspective, can be merged as-is, we can always change it later.